Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
I don't have a strong preference here -> Alpine Linux is become the de facto standard in Docker, and it's very small, so I'm very happy with it
I'm not really confident on "java-oracle" -> is it done by Oracle? I have a very bad experience with their downloads, with Netbeans included or browser add-ons installed without a clear notice